components/KeyboardScrollView.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { KeyboardAvoidingView, Platform,ScrollView } from 'react-native'
+
+import { ButtonSubmit } from './ButtonSubmit'
+import { styles } from './KeyboardScrollView.styles'
+
+export const KeyboardScrollView = (props) => {
+  const stylesOverride = props.style
+    ? {...styles, ...props.style}
+    : styles
+
```

Comment:
is there an advantage to writting it like this over `[styles, props.style]` like we do elsewhere? it's the same right or am I being dumb?